clusterDeployer.py

```diff
@@ -69,6 +73,16 @@ def setup_dhcp_entry(h: host.Host, cfg: NodeConfig) -> None:
     h.run_or_die(cmd)
 
 
+def match_regex(version_cluster_config: str) -> str:
```

Comment:
This function is a good start. But the function name needs work to clearly state what the intent of this function will do. Think of every function as a type of API. If a developer needs to reuse your code what will they look for.